catalyst_voices/lib/widgets/drawer/voices_drawer_space_chooser.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:catalyst_voices/widgets/drawer/voices_drawer.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:flutter/material.dart';
+
+class VoicesDrawerSpaceChooser extends StatelessWidget {
+  final Space currentSpace;
+  final ValueChanged<Space> onChanged;
+
+  const VoicesDrawerSpaceChooser({
+    super.key,
+    required this.currentSpace,
+    required this.onChanged,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return VoicesDrawerChooser<Space>(
+      items: Space.values,
+      selectedItem: currentSpace,
+      onSelected: (value) => onChanged(value),
```

Comment:
simpler `onSelected: onChanged` reads better?

Reply:
btw, can we just write

```
onSelected: onChanged
```
?